dvc/utils/__init__.py

```diff
@@ -183,16 +183,26 @@ def remove(path):
             raise
 
 
-def to_chunks(l, jobs):
-    n = int(math.ceil(len(l) / jobs))
+def to_chunks(l, num_chunks=None, chunk_size=None):
+    if (num_chunks and chunk_size) or (not num_chunks and not chunk_size):
+        from dvc.exceptions import DvcException
 
-    if len(l) == 1:
-        return [l]
+        raise DvcException(
```

Comment:
Probably the `ValueError` is more appropriate here. Would unit test it though :)